worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -822,6 +822,8 @@ namespace RTC
 			this->keyFrameForTsOffsetRequested = false;
 		}
 
+		bool marker{ false };
```

Comment:
This is always false and it's never set to true, so it would be better if we call `packet->ProcessPayload(this->encodingContext.get(), /*marker*/ false);` in lines below, right?

Reply:
@ibc ProcessPayload needs a boolean& and not a boolean

Reply:
> EDIT: VP8.cpp does not use the marker parameter, so yes this will always be false

Simulcast is not just about VP8 (we also support H264) and SinulcastConsumer is supposed to be agnostic. I don't understand if this is then ok or not.

Reply:
> @ibc  ProcessPayload needs a boolean& and not a boolean

Can you clarify what you mean please?

Reply:
The function signature requires a reference to a boolean value. This value will be set by the implementation if a market bit is observed to be true. Not all codec operations will require using this bit. In the case of VP9/SVC we need to know if this bit is set.